src/libraries/System.Runtime/tests/System/Reflection/NullabilityInfoContextTests.cs

```diff
@@ -451,16 +451,34 @@ public void GenericNotNullConstraintTest(string fieldName, NullabilityState read
             Assert.Equal(type, nullability.Type);
         }
 
-        public static IEnumerable<object[]> GenericStructConstraintTestData()
+        public static IEnumerable<object[]> GenericNotnullConstraintPropertiesTestData()
+        {
+            yield return new object[] { "PropertyNullable", NullabilityState.NotNull, NullabilityState.NotNull, typeof(string) };
```

Comment:
Shouldn't "PropertyNullable" be `NullabilityState.Null`?

Reply:
I thought it should be NotNull because `T` constrained to `notnull`, but after looking it again, you are right, Generic type `T`'s nullability should not override the property's `?` 
```cs
    internal class GenericTestConstrainedNotNull<T> where T : notnull
    {
        public T PropertyNullableEnabled { get; set; } = default!;
        public T? PropertyNullable { get; set; }
    }
```